guides/common/modules/proc_configuring-host-based-access-control-for-freeipa-users-logging-in-to-project.adoc

```diff
@@ -32,7 +32,12 @@ endif::[]
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
+ifdef::foremanctl[]
+# foremanctl deploy --pam-service foreman-prod
```

Comment:
Now that I'm seeing that with fresh eyes I wonder if we should come up with a more self-descriptive name for the parameter. `--external-auth-pam-service` maybe?

Reply:
I've said in various places already, but this is a great example of where docs force you to see your own code through the eyes of a user.